modules/app.js

```diff
@@ -11,6 +12,7 @@ const addBtn = document.querySelector('.add');
 
 storage = JSON.parse(localStorage.getItem('books')) || [];
 
+//  ADD BOOK HERE
 function addBook(book, title, author) {
   book.title = title;
   book.author = author;
```

Comment:
- [ ] As in the project requirements, **we have to implement ES6 modules in our app. We are expected to practice splitting our code in the `app.js` into small chunks of code with specific functionality in their separate files called modules**. Please try to make more modules from this app.js **For example you can make a js file to contain this function to add the book then later you import it back into the app.js**. You can also think about doing the same on the **remove book function**. For all of this, make sure that your app is still working well. 😉